source/math/FloatVectorList.ooc

```diff
@@ -224,7 +224,7 @@ FloatVectorList: class extends VectorList<Float> {
 		result := This new(size)
 		factor := 1.0f / (sqrt(2.0f * Float pi) * sigma)
 		for (i in 0 .. size)
-			result add(factor * pow(Float e, -0.5f * ((i - (size - 1.0f) / 2.0f) squared()) / (sigma squared())))
+			result add((factor * pow(Float e, -0.5f * ((i - (size - 1.0f) / 2.0f) squared()) / (sigma squared()))) as Float)
```

Comment:
What part of this formula turned everything into something other than a `Float`? Does `Double` need casting into `Float` before it can be added to a `FloatVectorList`?